doc/source/user/troubleshooting-importerror.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Can you add an rst comment here explaining why this is an orphan? One reason answer would be "the main webpage links here" or "the url to this page appears in an error message.

Reply:
Good suggestion. In 50811dc.